### DISCUSSION TITLE
mention the mesh tools plugin in the readme

### DISCUSSION THREAD

**Discussion by rijkvanmanen** [5axes/NameIt#11] [Ideas]
I think this plugin is especially useful in combination with the Mesh Tools plugin that allows you to rename objects. Maybe mention that plugin in the readme?

**Comment by 5axes** [comment]
Yes , or also add a similar function in the plugin. Something different ( Ie : the mesh tools allow you to modify the name only one by one, could be interresting to do it on a global selection)

**Reply by 5axes** [reply]
Add a Rename Model**s** function V1.6.0

![image](https://user-images.githubusercontent.com/11015345/177128044-35ec61c4-81f2-4da8-aab2-918f204bbd02.png)

**Comment by 5axes** [comment]
Modification compare to the Mesh tool Function : 
You can rename several model in the same times

In case of several modifications  the first Item will start from (0)  -> can be usefull for the function **Add Number From Part** Ie : https://github.com/5axes/NameIt/issues/4#issuecomment-1159746200

![image](https://user-images.githubusercontent.com/11015345/177132679-5c8fce21-e2a0-4739-87d7-77b39e5f2218.png)

**Comment by rijkvanmanen** [comment]
Nice! Thank you. really appreciate it.
